jans-cedarling/cedarling/src/models/log_entry.rs

```diff
@@ -106,13 +106,24 @@ pub struct AuthorizationLogInfo {
 }
 
 /// Cedar-policy decision of the authorization
-#[derive(Debug, Clone, PartialEq, Copy, serde::Serialize, serde::Deserialize)]
+#[derive(Debug, Clone, PartialEq, Eq, Copy, serde::Serialize, serde::Deserialize)]
 #[serde(rename_all = "UPPERCASE")]
 pub enum Decision {
+    /// Determined that the request should be allowed
     Allow,
+    /// Determined that the request should be denied.
     Deny,
 }
 
+impl ToString for Decision {
```

Comment:
This has a bit of a risk of becoming the default string representation for the `Decision`. But that's always tricky with languages like python that don't have built-in sum types.

Reply:
I tried to create Decision from python and got an error
```
Traceback (most recent call last):
  File "d:\________\jans\jans-cedarling\bindings\cedarling_python\example.py", line 100, in <module>
    q = Decision()
        ^^^^^^^^^^
TypeError: No constructor defined for Decision
```
So we have no problems. Because `Decision` python class can be constructed only from save rust part of code.

If I understand corrrectly.

Reply:
And we have no default implementation for this structure

Reply:
Could you suggest an approach for python?

Reply:
What's the generally accepted way in Python to deal with small sum types? Looks to me several people here are better at python than I am.

In general, each language binding will need to deal with this problem (small sum types) in its own language-specific way. So probably better for that mapping to be done in `cedarling_python::authorize::Decision` rather than `cedarling::bindings::Decision`

Reply:
Resolved this in messaging chat.